tree/treeplayer/inc/ROOT/TDFInterface.hxx

```diff
@@ -728,7 +729,7 @@ public:
    /// This action is *lazy*: upon invocation of this method the calculation is
    /// booked but not executed. See TResultPtr documentation.
    template <typename T, typename COLL = std::vector<T>>
-   TResultPtr<typename TDFDetail::ColType_t<T, COLL>> Take(std::string_view column = "")
+   TResultPtr<COLL> Take(std::string_view column = "")
```

Comment:
iiuc now `Take<TVec<T>>` returns a `TResultPtr<TVec<T>>` instead of a `TResultPtr<std::vector<T>>` like before.
this change in return type probably deserves its own commit.

lifetimes are ok right? we will never return adopting `TVec`s I guess..?

Reply:
I thought about this. We may be so efficient (0 copies) that we may end up with an adopting TVec. I'll force the realloc explicitly.